idp-service/README.md

```diff
@@ -1,8 +1,63 @@
 ## Identity Provider Service
 
+IdP OIDC service, provides endpoints to 
+1. Generate auth-code 
```

Comment:
Do we mean here to invoke authorize flow ?

Reply:
Yes, Still need to work on the readme file